hooks/pre-commit

```diff
@@ -25,7 +25,7 @@ validate_version() {
   fi
 }
 
-diff=$(git diff --cached --name-only | grep '\.py$' || true)
+diff=$(git diff --cached --diff-filter ACMRTUXB --name-only | grep '\.py$' || true)
```

Comment:
This is necessary to get past the pre commit hook if a file gets deleted. `black` will try to run on a deleted file since it shows up in the `git diff`, which results in an error. This filter selects all changes except file deletion (thanks @harupy!)